src/components/pages/Signup/Signup.js

```diff
@@ -0,0 +1,161 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+// import { reach } from 'yup'
+// import schema from '../../../utils/validation/formSchema'
+import { Form, Input, Button, Radio } from 'antd';
+
+const initialFormValues = {
+  email: '',
+  password: '',
+  firstName: '',
+  lastName: '',
+};
+
+const initialFormErrors = {
+  email: '',
+  password: '',
+  firstName: '',
+  lastName: '',
+};
+
+const Signup = () => {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [firstName, setFirstName] = useState('');
+  const [lastName, setLastName] = useState('');
+
+  const [formValues, setFormValues] = useState(initialFormValues);
+  const [formErrors, setFormErrors] = useState(initialFormErrors);
+
+  const [value, setValue] = React.useState(1);
+
+  const postNewAccount = async newAccount => {
+    try {
+      const response = await axios.post(' ', newAccount);
```

Comment:
I'm assuming this is also a stub to represent where we'll be putting the actual call once we're ready to do so?